### PR TITLE
lizadams adds hometown of Cary, NC

### PR DESCRIPTION
**Pull request by lizadams**
- can you mention people who are not collaborators on this project but who are on github?
- is anyone interested in parks?

### CODE REVIEW

File: lizadams-cary.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Cary, NC
```
Comment (WilliamBonner) [inline lizadams-cary.md:1]: @lizadams How is the weather in Dary?

File: lizadams-cary.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+### Restaurants
+- La Farm
```
Comment (WilliamBonner) [inline lizadams-cary.md]: @lizadams What is the food like in La Form?

File: lizadams-cary.md (hunk 1 above)
Comment (lizadams) [inline lizadams-cary.md:1]: It is beautiful - 80 degrees!  @WilliamBonner - How is the weather in your hometown?

File: lizadams-cary.md (hunk 2 above)
Comment (lizadams) [inline lizadams-cary.md]: @WilliamBonner It is a bakery - and the food is yummy!

**Pull request merged**